samples/fabric/fabric-cli/src/data/credentials/Corda_Network/access-control.json

```diff
@@ -5,5 +5,17 @@
     "principalType": "certificate",
     "resource": "mychannel:simplestate:Read:*",
     "read": true
+  },
+  {
```

Comment:
Please make similar change for `Corda_Network2`, as well as for `credentials_docker`